pyccel/ast/c_concepts.py

```diff
@@ -146,7 +158,7 @@ def get_flat_expression_list(self):
     def expression(self):
         """ The list containing the literal strings and c macros
         """
-        return self._expression
+        return tuple(self._expression)
```

Comment:
Is this still needed? Isn't `self._expression` a tuple now?

Reply:
no it's not needed anymore I will remove it